Sources/ClientRuntime/Networking/Http/Middlewares/RequestBody/EventStreamBodyMiddleware.swift

```diff
@@ -17,13 +17,16 @@ public struct EventStreamBodyMiddleware<OperationStackInput,
 
```

Comment:
Instead of getting the `requestEncoder` from middleware context and passing it along to `DefaultMessageEncoderStream`, the middleware now takes in a `marshalClosure` at initialization and passes it along to `DefaultMessageEncoderStream`.